### DISCUSSION TITLE
there is a issue with my canvas, if i am opening layers tab the gjs-select moved to left.

### DISCUSSION THREAD

**Discussion by adarshsinghgaian** [GrapesJS/grapesjs#5495] [General]
![Screenshot from 2023-11-02 13-40-18](https://github.com/GrapesJS/grapesjs/assets/128805480/5ae96c9a-23e7-4477-bafa-6b9c2e0548f6)

**Comment by artf** [comment]
You can use [`editor.refresh()`](https://grapesjs.com/docs/api/editor.html#refresh)
